docker/policies/latest_tag.rego

```diff
@@ -60,11 +71,13 @@ image_tags[[img, tag]] {
 	[img, tag] = parse_tag(bare_image_name)
 }
 
-# fail_latest is true if image is not scratch and
-# tag is latest.
+# fail_latest is true if image is not scratch
+# and image is not an alias
+# and tag is latest.
 fail_latest[img] {
 	[img, tag] := image_tags[_]
 	img != "scratch"
+	not is_alias(img)
```

Comment:
I think it makes more sense to move this line into the `parse_tag` function that adds `latest` as default. This function should add the tag only if not alias. WDYT?

Reply:
`parse_tag` is intended to parse image name. It's not clear what should be returned if  image name is alias. So I would keep it as is